memo.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require 'json'
+
+require_relative 'memo_class'
+
+helpers do
+  def h(text)
+    Rack::Utils.escape_html(text)
+  end
+
+  def params_validation(path, title, content)
+    redirect to(path) if title.empty? || content.empty?
+  end
+
+  def all_memos
+    File.open('memos.json') do |file|
+      FileTest.empty?('memos.json') ? nil : JSON.parse(file.read, symbolize_names: true)
```

Comment:
```suggestion
      FileTest.empty?(file) ? [] : JSON.parse(file.read, symbolize_names: true)
```

* `memos.json` が２回でてくるので `file` 引数を使うとよさそうです。
* `nil` を返す代わりに `[]` と空の配列を返すと `index.erb` で `@memos.each` が必ず配列を期待できるのでオススメです。

Reply:
ご指摘の通り修正しました。
後述の`POST`メソッド内で`all_memos`メソッドが使える様になったので、グッとコードが綺麗になりました！
https://github.com/Kassy0220/memo_app/blob/60082615e7dfc0bea9ac794a920359c13ff61074/app.rb#L22

空の配列を返す様になったことで、`index.erb`でぼっち演算子を使う必要がなくなったため、削除しています。
https://github.com/Kassy0220/memo_app/blob/60082615e7dfc0bea9ac794a920359c13ff61074/views/index.erb#L9